### ISSUE TITLE
Support stylelint 11

### ISSUE THREAD

**Issue by limonte** [cahamilton/stylelint-config-property-sort-order-smacss#17]
After upgrading stylelint to 11, I'm getting this warning:

```
warning "stylelint-config-property-sort-order-smacss@5.0.0" has incorrect peer dependency "stylelint@^10.0.0".
```

**Comment by Vages** [comment]
Are you working on this, @cahamilton? If not, I could give it a go; just say the word.

**Comment by JemarJones** [comment]
Any update on this?

**Comment by cahamilton** [comment]
Hey guys, thanks for being patient. Let me take a look at this over the next few days. Need to re-familiarise myself with it all and update a couple pieces. Looks like stylelint@12.0.0 is out now as well, so will cover that in this update

**Comment by cahamilton** [comment]
@limonte @Vages @JemarJones Hey guys. Threw a couple changes together before and published a couple minor updates to this package on npm. Changes should allow this to work up to the current release of stylelint (12.0.0). Let me know if you have any problems 👍
